### PR TITLE
README.md:add info about using PKG_CONFIG_PATH when geany is already …

### PR DESCRIPTION
**Pull request by andy5995**
…installed

### CODE REVIEW

**Comment by andy5995** [comment]
The reason I think this would be a worthwhile change is because I just ran into that situation when trying to build EC for the first time. Took me a half hour to remember how to do it xD

File: README.md [hunk 1]
```diff
@@ -24,6 +27,12 @@ This is an [EditorConfig][] plugin for [Geany][].
 
         make EDITORCONFIG_PREFIX=/usr
 
+    If you've installed geany from source, you'll need to preface the
+    `make` command above with the PKG_CONFIG_PATH variable, and set it
+    to the prefix used when you built geany. Example:
```
Comment (xuhdev) [inline README.md]: Thanks, but this paragraph is somewhat confusing. Could you add some explanation on what one would need so?

File: README.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ This is an [EditorConfig][] plugin for [Geany][].
 
         apt-get install libgtk2.0-dev libgtk-3-dev intltool geany
 
+    **Note** If you've already installed geany from source, you can
+    skip installing them from your repository.
```
Comment (xuhdev) [inline README.md]: How about making this clear above instead of here? We can get other dependencies also covered, and make clear that the commands here are merely an example (there are certainly many other ways other than "installing from source"). For example,

> Install the [EditorConfig C Core][]. **For example,** if you are using Debian/Ubuntu, you can install it by running the following command:

> You also need to install geany and GTK development files (GTK2 or GTK3 depending on your system). **For example**, this could be done by executing the following command if you are using Debian/Ubuntu:

File: README.md (hunk 1 above)
Comment (andy5995) [reply README.md]: Sure, I can try to be more clear. My situation, for example, is that I do not have geany installed from my Debian repo. I built geany from the git source. When building geany, I used `./configure --prefix=$HOME/.local`. So in order for pkgconfig to find the .pc file, I needed to adjust the pkg config search path so your Makefile could find the geany pc file and import the CFLAGS. The dependencies I needed to build geany, I did get those from my repo.

Basically if someone has built geany from source, that implies they already have all the required deps installed (or should know how to re-obtain them).

Comment (xuhdev) [reply README.md]: OK, thanks. Please try to write a bit more -- we do not fear verbosity as long as it is clear and not confusing :).

Also, is it correct to say that the example below also needs to be updated to match the value of EDITORCONFIG_PREFIX to that of PKG_CONFIG_PATH?

Comment (andy5995) [reply README.md]: > OK, thanks. Please try to write a bit more -- we do not fear verbosity as long as it is clear and not confusing :).
> 

Ok, fair enough :)

> Also, is it correct to say that the example below also needs to be updated to match the value of EDITORCONFIG_PREFIX to that of PKG_CONFIG_PATH?

Looking at the Makefile ....
```
ifndef EDITORCONFIG_PREFIX
        EDITORCONFIG_PREFIX = /usr/local
endif
TARGET_ARCHITECTURE= $(shell cc -dumpmachine)

CC = cc
CFLAGS = -Wall -Wextra -O2 -g -fPIC `pkg-config --cflags geany` -I${EDITORCONFIG_PREFIX}/include
LDFLAGS = -shared `pkg-config --libs geany` -L ${EDITORCONFIG_PREFIX}/lib -leditorconfig
```

I think the EC prefix is for the EC installation. So assuming most users install that from their repo, it's going to be `/usr` and doesn't have anything to do with the Geany installation. the info for Geany is retrieved separately by the `pkg-config geany --cflags` and `--libs` directives.

For example, on my system, when I run it from the command line:

andy@oceanus:~$ PKG_CONFIG_PATH=$HOME/local/lib/pkgconfig  pkg-config geany --cflags
-DGTK -pthread -I/home/andy/local/include/geany -I/home/andy/local/include/geany/tagmanager -I/home/andy/local/include/geany/scintilla -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0 -I/usr/include/cairo -I/usr/include/libdrm -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/pango-1.0 -I/usr/include/fribidi -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/uuid -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include

So you see all the compiler flags that get added in that case.

File: README.md (hunk 2 above)
Comment (andy5995) [reply README.md]: I'll have another look at the README tomorrow and give some thought to re-phrasing it, or moving things a little.

**Comment by xuhdev** [comment]
Thanks for your contribution!

**Comment by andy5995** [comment]
You're welcome :)
